app/electron/main.js

```diff
@@ -16,25 +19,59 @@ const Store = require("secure-electron-store").default;
 const ContextMenu = require("secure-electron-context-menu").default;
 const path = require("path");
 const fs = require("fs");
+const log = require("electron-log");
+
 const Protocol = require("./protocol");
 const MenuBuilder = require("./menu");
 
 const isDev = process.env.NODE_ENV === "development";
 const port = 40992; // Hardcoded; needs to match webpack.development.js and package.json
 const selfHost = `http://localhost:${port}`;
 
+// Setup file logging
+
+log.transports.file.level = "info";
+log.transports.file.file = `${__dirname}logger.log`;
+
 // Keep a global reference of the window object, if you don't, the window will
 // be closed automatically when the JavaScript object is garbage collected.
 let win;
 let menuBuilder;
 
 async function createWindow() {
   if (isDev) {
-    await installExtension([REDUX_DEVTOOLS, REACT_DEVELOPER_TOOLS]);
-    /*  !TODO LOG
-      .then((name) => console.log(`Added Extension:  ${name}`))
-      .catch((err) => console.log("An error occurred: ", err));
-    */
+    await installExtension([REDUX_DEVTOOLS, REACT_DEVELOPER_TOOLS])
+      .then((name) => log.info(`Added Extension:  ${name}`))
+      .catch(
+        log.catchErrors({
+          showDialog: true,
+          onError(error, versions, submitIssue) {
+            dialog
+              .showMessageBox({
+                title: "An error occurred",
+                message: error.message,
+                detail: error.stack,
+                type: "error",
+                buttons: ["Ignore", "Report", "Exit"],
+              })
+              .then((result) => {
+                if (result.response === 1) {
+                  submitIssue("https://github.com/djunicode/ergo/issues/new", {
+                    title: `Error report for ${versions.app}`,
+                    body:
+                      `Error:\n\`\`\`${error.stack}\n\`\`\`\n` +
+                      `OS: ${versions.os}`,
+                  });
```

Comment:
Are you sure this is how error handling is to be set up ? @namandangi  can you once verify this? The impression that I am getting from electron-log is that this should be a global config and somehow it will handle uncaught errors, but not sure : check https://github.com/megahertz/electron-log/blob/master/docs/catch.md